tests/src/test/scala/spire/math/MultivariatePolynomialTest.scala

```diff
@@ -0,0 +1,341 @@
+package spire.math
```

Comment:
The test case looks good. Property-based testing, and a few hand-written cases that show usage examples.